hack/update-codegen.sh

```diff
@@ -43,11 +43,10 @@ setgen=$(kube::util::find-binary "set-gen")
 # update- and verify- scripts.
 ${clientgen} "$@"
 ${clientgen} -t "$@"
-
+${clientgen} --clientset-name="release_1_3" --input="api/v1,extensions/v1beta1"
```

Comment:
Shouldnt we add  `autoscaling/v1,batch/v1` here?
What about alpha groups? apps and metrics.